state/executor.go

```diff
@@ -202,6 +209,23 @@ func (t *Transition) Receipts() []*types.Receipt {
 
 var emptyFrom = types.Address{}
 
+func (t *Transition) WriteFailedReceipt(txn *types.Transaction) {
+	logs := t.state.Logs()
+
+	receipt := &types.Receipt{
+		CumulativeGasUsed: t.totalGas,
+		TxHash:            txn.Hash,
+	}
+	receipt.Logs = logs
```

Comment:
Lines 213 and 219 can be a single line inside the struct initialization block.

Reply:
fixed 14b2e7d2ac895d4423be57434df9e31d66942643